sim/paladin/retribution/retribution_test.go

```diff
@@ -111,6 +192,8 @@ var Phase1RetTalents = "--05230051"
 var Phase2RetTalents = "--532300512003151"
 var Phase2ShockadinTalents = "55050100521151--"
 var Phase3RetTalents = "500501--53230051200315"
+var Phase45RetTalents = "500501-503-52230351200315"
```

Comment:
Phase 45 :POG: lol jk this is fine

Reply:
the year is 2030 and still no sign of classic plus but we have SoD phase 45 at least